WordPress/Classes/ViewRelated/Blog/Blogging Prompts/BloggingPromptCoordinator.swift

```diff
@@ -20,7 +20,6 @@ import UIKit
         case actionSheetHeader
         case promptNotification
         case promptStaticNotification
-        case unknown
```

Comment:
The remove of this case generates the `Default will never be executed` warning in `var editorEntryPoint`.

Reply:
Good catch – I fixed it in one of the newer PRs.